src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -215,28 +205,65 @@ export class MeetingManager implements AudioVideoObserver {
   }
 
   async join({
-    meetingInfo,
-    attendeeInfo,
-    deviceLabels = DeviceLabels.AudioAndVideo,
+    meetingSessionConfig,
+    deviceLabels,
     eventReporter,
     meetingManagerConfig,
   }: MeetingJoinData) {
+    if (deviceLabels === null || deviceLabels === undefined) {
+      deviceLabels = DeviceLabels.AudioAndVideo;
+    }
     if (meetingManagerConfig) {
       this.meetingManagerConfig = meetingManagerConfig;
     }
-    this.configuration = new MeetingSessionConfiguration(
-      meetingInfo,
-      attendeeInfo
-    );
+    if (meetingSessionConfig === null) {
+      throw new Error('Meeting Session Configuration parameter cannot be null');
+    }
 
-    this.meetingRegion = meetingInfo.MediaRegion;
-    this.meetingId = this.configuration.meetingId;
+    this.meetingSessionConfig = meetingSessionConfig;
+    this.meetingId = this.meetingSessionConfig.meetingId;
 
     if (eventReporter) {
       this.eventReporter = eventReporter;
     }
 
-    await this.initializeMeetingSession(this.configuration, deviceLabels);
+    // We are assigning the values from the `meetingManagerConfig` to preserve backward compatibility.
+    // Please use `this.meetingManagerConfig` for any values going forward.
+    if (this.meetingManagerConfig.simulcastEnabled) {
+      meetingSessionConfig.enableUnifiedPlanForChromiumBasedBrowsers = true;
```

Comment:
You can remove this part as it is always true.

Reply:
You can see here how simulcastEnabled just maps to an actual property in session configuration ;)

Reply:
Done